README.md

```diff
@@ -181,7 +181,7 @@ Run `npx gulp api` which will build the documentation files and place them in th
 # Pull Requests
 
 I'd be happy to review any pull requests that may better the Konva project,
-in particular if you have a bug fix, enhancement, or a new shape (see `src/shapes` for examples). Before doing so, please first make sure that all of the tests pass (`gulp lint test`).
```

Comment:
I believe gulp lint test are no longer available? Or did I missed something?

Reply:
gulp is here. But I don't think that `gulp lint test` command is the last one to use. it should be just `npm run test`

Reply:
Good, so I'll keep this change 👍